scripts/get-coderabbit-comments.sh

```
@@ -0,0 +1,631 @@
+        .metadata.resolution_rate = (if (.comments | length) > 0 then (($resolved_list | length) * 100 / (.comments | length)) | floor else 0 end)
+    ' "$comments_file" > "$comments_file.tmp" && mv "$comments_file.tmp" "$comments_file"
+}
+
+# Handle resolution commands
+if [ "$COMMAND" = "resolve" ]; then
+    # Check if multiple comment IDs are provided (comma-separated)
+    if [[ "$COMMENT_ID" == *","* ]]; then
+        save_multiple_resolutions "$COMMENT_ID"
+        update_local_comments_state
+        # Count the number of resolved comments
+        local count
```

Shellcheck (0.11.0)

[error] 'local' is only valid in functions.

(SC2168)